<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextSettings</class>
 <widget class="QWidget" name="TextSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>308</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QFontComboBox" name="font">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="size">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>72.000000000000000</double>
       </property>
       <property name="value">
        <double>12.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="halign">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="toolTip">
        <string>Horizontal alignment</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="valign">
       <property name="toolTip">
        <string>Vertical alignment</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="topSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="widgets::ColorButton" name="btnBackground">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Background color</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="color" stdset="0">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="setAlpha" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::ColorButton" name="btnTextColor">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Text color</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="setAlpha" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bold">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Bold</string>
       </property>
       <property name="icon">
        <iconset theme="format-text-bold">
         <normaloff>theme:format-text-bold.svg</normaloff>theme:format-text-bold.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="italic">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="toolTip">
        <string>Italic</string>
       </property>
       <property name="icon">
        <iconset theme="format-text-italic">
         <normaloff>theme:format-text-italic.svg</normaloff>theme:format-text-italic.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="underline">
       <property name="toolTip">
        <string>Underline</string>
       </property>
       <property name="icon">
        <iconset theme="format-text-underline">
         <normaloff>theme:format-text-underline.svg</normaloff>theme:format-text-underline.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="strikethrough">
       <property name="toolTip">
        <string>Strikethrough</string>
       </property>
       <property name="icon">
        <iconset theme="format-text-strikethrough">
         <normaloff>theme:format-text-strikethrough.svg</normaloff>theme:format-text-strikethrough.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="render">
       <property name="toolTip">
        <string>Rendering</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="content">
     <property name="placeholderText">
      <string>Annotation content</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="widgets::GroupedToolButton" name="protectButton">
       <property name="groupPosition">
        <enum>widgets::GroupedToolButton::GroupLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::GroupedToolButton" name="mergeButton">
       <property name="groupPosition">
        <enum>widgets::GroupedToolButton::GroupCenter</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::GroupedToolButton" name="deleteButton">
       <property name="groupPosition">
        <enum>widgets::GroupedToolButton::GroupRight</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="creatorLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">Creator</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>widgets::ColorButton</class>
   <extends>QToolButton</extends>
   <header>desktop/widgets/colorbutton.h</header>
   <slots>
    <signal>colorChanged(QColor)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>widgets::GroupedToolButton</class>
   <extends>QToolButton</extends>
   <header>desktop/widgets/groupedtoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>italic</sender>
   <signal>toggled(bool)</signal>
   <receiver>content</receiver>
   <slot>setFontItalic(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>size</sender>
   <signal>valueChanged(double)</signal>
   <receiver>content</receiver>
   <slot>setFontPointSize(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTextColor</sender>
   <signal>colorChanged(QColor)</signal>
   <receiver>content</receiver>
   <slot>setTextColor(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>underline</sender>
   <signal>toggled(bool)</signal>
   <receiver>content</receiver>
   <slot>setFontUnderline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
